core/src/main/java/org/jruby/RubyInstanceConfig.java

```diff
@@ -424,9 +424,18 @@ public InputStream getScriptSource() {
     private static InputStream findScript(InputStream is) throws IOException {
         StringBuilder buf = new StringBuilder(64);
         BufferedReader br = new BufferedReader(new InputStreamReader(is));
-        String currentLine = br.readLine();
-        while (currentLine != null && !isRubyShebangLine(currentLine)) {
-            currentLine = br.readLine();
+
+        boolean foundRubyShebang = false;
+        String currentLine;
+        while((currentLine = br.readLine()) != null) {
```

Comment:
_One tiny thing_: This `while` statement has different whitespace.

It needs a space after the `while` keyword to keep it the same.

Same goes for the `if` statements below.

(Thanks for working to imrpove JRuby!)

Reply:
Oops, you are totally right, fixed!